src/intrinsics/SetPrototype.mts

```diff
@@ -162,6 +169,59 @@ function SetProto_values(args, { thisValue }) {
   return Q(CreateSetIterator(S, 'value'));
 }
 
+/** http://tc39.es/ecma262/#sec-set.prototype.union */
```

Comment:
is that in the main spec now? if not, a link to the proposal spec will be better!